unicodetools/src/main/java/org/unicode/props/IndexUnicodeProperties.java

```diff
@@ -693,17 +676,48 @@ class IndexUnicodeProperty extends UnicodeProperty.BaseProperty {
             }
         }
 
+        @Override
         protected UnicodeMap<String> _getUnicodeMap() {
+            var raw = _getRawUnicodeMap();
```

Comment:
Ah, looks like you are leaving them in the internal map, but then refashioning the map. Would be better to do this once in the internal map instead, probably.

Reply:
But then we are back to high memory usage.

Reply:
For additional context: I started looking at that while trying to make the versioned JSPs use less memory by storing only differences for earlier versions; right now the staging JSPs have a container with more memory than they used to so they can hold the whole UCD history.

Reply:
(And in fact that container eventually runs out of memory while loading Unicode Version 5.0. The fact that it takes it about twenty hours to reach Unicode 5.0 when on any of my machines it loads all history in 15ish minutes raises questions as to the cultivar of potato we run the JSPs on, but there is room for performance improvement here both in precomputation and memory usage.)